tests/Extractor/HttpClientTest.php

```diff
@@ -290,8 +288,25 @@ public function testTimeout()
         $handler = new TestHandler();
         $logger->pushHandler($handler);
 
-        $guzzle = new GuzzleClient(['defaults' => ['timeout' => 2]]);
-        $adapter = new GuzzleAdapter($guzzle);
+        // find with adapter is installed
+        if (class_exists('Http\Adapter\Guzzle6\Client')) {
```

Comment:
it's almost like the discovery of php-http http://docs.php-http.org/en/latest/discovery.html but if you want configure the timeout, I think there is no other way